gleam_stdlib/src/gleam_stdlib.erl

```diff
@@ -90,3 +90,13 @@ parse_float(String) ->
     _ ->
       {error, nil}
   end.
+
+compare_strings(Lhs, Rhs) ->
+  if
+    Lhs == Rhs ->
+      0;
+    Lhs < Rhs ->
+      -1;
+    true ->
+      1
```

Comment:
Looks good!

We could return `lt` `eq` and `gt` rather than ints here to avoid the extra conversion on the Gleam side.